src/components/views/settings/tabs/user/PreferencesUserSettingsTab.js

```diff
@@ -48,6 +48,7 @@ export default class PreferencesUserSettingsTab extends React.Component {
         'showRedactions',
         'enableSyntaxHighlightLanguageDetection',
         'expandCodeByDefault',
+        `scrollToBottomOnMessageSent`,
```

Comment:
Matching quote styles in the area is important to us:
```suggestion
        'scrollToBottomOnMessageSent',
```

Reply:
I've done edits to this file before but this time my brain just decided to do a weird mistake :joy: